pkg/utils/jwks.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+	"encoding/json"
+	"reflect"
+)
+
+// Checks if two string parameters represent equal json web key sets. false is
+// returned if the two jwks do not have equivalent values, or if there is an
+// error processing the expected fields of either parameter.
+func JwksEqual(
+	jwksStrA string,
+	jwksStrB string,
+) bool {
+	var jwksA, jwksB struct {
+		Keys []struct {
```

Comment:
What about `x5c` and `x5t`?

Is there a a third party comparison tool that we can use? trying not to re-invent the wheel

Reply:
updated to use https://pkg.go.dev/github.com/MicahParks/jwkset#JWKSMarshal